.github/workflows/pr-linter.yml

```diff
@@ -83,6 +108,8 @@ jobs:
   pr-clang-tidy:
     name: Clang-Tidy
     runs-on: ubuntu-latest
+    needs: pr-changes
```

Comment:
This check is x64 specific currently, so it needs to run only on cpu-x64 and gpu-intel related changes.